lib/digital_public_works_web/templates/project/_projects.html.eex

```diff
@@ -1,3 +1,6 @@
+<% is_admin = (@conn.assigns[:current_user] != nil) and @conn.assigns[:current_user].is_admin %>
 <%= for project <- @projects do %>
-<%= render "_project.html", project: project, conn: @conn %>
-<% end %>
\ No newline at end of file
+  <%= if project.is_public or is_admin do %>
```

Comment:
We could modify Projects.list_projects to accepted a User argument and filter out non-public projects if the user is not an admin.